source/includes/overview/_quickstart.md

```diff
@@ -174,6 +237,40 @@ with open(path, "rb") as fp:
 r.json()
 ```
 
+``` javascript
+const fs = require('fs/promises');
+const axios = require('axios');
+
+const filePath = '<FILE_PATH>'; // Replace with file path
+const bucketURL = '<BUCKET_URL>'; // Replace with bucket url
+const fileName = '<FILE_NAME>'; // Replace with file name
+const token = 'TOKEN'; // Replace with token value
+
+// Read file
+const file = await fs.readFile(filePath);
```

Comment:
I'm not sure about `fs.readFile`, but it might not be entirely equivalent to the Python example, since I think it reads the whole file in memory (same in Python as calling `data = open('myfile.zip').read()`) and passes it to the request, instead of streaming the file in chunks. Not sure what the equivalent is for NodeJS (if it's too complicated to adjust it's fine to shelve this and address later).

Reply:
It is possible, using [Streams](https://nodejs.org/api/stream.html). However, the file stream must be added to a form (using [FormData](https://www.npmjs.com/package/form-data)).

Updated the docs, thanks for the suggestion!